packages/wonder-blocks-dropdown/util/constants.js

```diff
@@ -34,3 +34,15 @@ export const SEPARATOR_ITEM_HEIGHT = 9;
 
 export const SEARCH_ITEM_HEIGHT =
     DROPDOWN_ITEM_HEIGHT + searchInputStyle.margin + searchInputStyle.marginTop;
+
+// The default labels that will be used by different components
+export const defaultLabels = {
```

Comment:
Now all the default labels are centralized in this file.

Reply:
Very nice!